### PR TITLE
fwf-3217: [Modified] created files for components

### PR DESCRIPTION
**Pull request by fahad-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3217
Issue Type: BUG/ FEATURE

# Changes
1. Created separate SCSS files for components 
2.  Removed  unused classes 
3.  Updated bootstrap classes and variables

### CODE REVIEW

**Review by Bonymol-aot — APPROVED** [review]

File: forms-flow-theme/scss/_button.scss [hunk 1]
```diff
@@ -22,4 +22,112 @@
 
 .btn-link:hover {
     text-decoration: underline !important;
+}
+
+.btn-primary { 
+    background-color: var(--button-primary-background);
```
Comment (shuhaib-aot) [inline forms-flow-theme/scss/_button.scss:28]: without over riding the class can we use direct variable over ride @Bonymol-aot @Josephalexantony-aot @fahad-aot

File: forms-flow-theme/scss/_typography.scss [hunk 2]
```diff
@@ -1,9 +1,70 @@
...
-
+$h4-font-size:1.275rem ;
+$h5-font-size : 1rem ;
 
 .input-group-text {
-    color: $gray-800 !important;
+  color: $gray-800 !important;
+}
+
+.text-primary {
```
Comment (shuhaib-aot) [inline forms-flow-theme/scss/_typography.scss:10]: is this really needed ?

Comment (fahad-aot) [reply forms-flow-theme/scss/_typography.scss:10]: Yes. Bootstrap text-primary color and formsflow text-primary color are different.

File: forms-flow-theme/scss/_button.scss (hunk 1 above)
Comment (Josephalexantony-aot) [reply forms-flow-theme/scss/_button.scss:28]: @shuhaib-aot there is no variable like $btn-primary-bg in bootstrap. still in this case as we already override the primary, that should reflect here as well.

File: forms-flow-theme/scss/_typography.scss (hunk 2 above)
Comment (fahad-aot) [reply forms-flow-theme/scss/_typography.scss]: updated . By overriding the primary color variable, as all Bootstrap components that use the primary color will update automatically. So removed classes which is related to primary color .

File: forms-flow-theme/scss/_button.scss (hunk 1 above)
Comment (fahad-aot) [reply forms-flow-theme/scss/_button.scss]: Updated.

**Pull request merged**